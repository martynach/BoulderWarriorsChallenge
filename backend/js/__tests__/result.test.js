const Result = require('./../Result');
const result = new Result();
const path = require("path");
const filepath = path.join(__dirname, './test_data/results.json');
result.filepath = filepath;


test('Get results for meeting with id=1', async () => {
    expect.assertions(1);
    const actual = await result.getResults(1);
    expect(actual).toMatchSnapshot();
});