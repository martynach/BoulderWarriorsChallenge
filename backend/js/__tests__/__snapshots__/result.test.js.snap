// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get results for meeting with id=1 1`] = `
Object {
  "boulders": Array [
    Object {
      "bonus": Array [
        1,
        2,
        3,
        4,
        5,
        6,
      ],
      "top": Array [
        1,
        2,
        3,
        4,
        5,
        6,
      ],
    },
    Object {
      "bonus": Array [
        1,
        2,
        3,
        4,
        5,
        6,
      ],
      "top": Array [
        1,
        2,
        3,
        4,
        5,
      ],
    },
    Object {
      "bonus": Array [
        1,
        2,
        3,
        4,
        5,
      ],
      "top": Array [
        2,
        3,
        4,
        5,
      ],
    },
    Object {
      "bonus": Array [
        2,
        3,
        4,
        5,
      ],
      "top": Array [
        3,
        4,
      ],
    },
    Object {
      "bonus": Array [
        2,
        3,
        4,
      ],
      "top": Array [
        3,
      ],
    },
  ],
  "meetingID": 1,
}
`;
